core/ppu_bg.go

```diff
@@ -158,8 +158,13 @@ func (ppu *PPU) bg4vofs(data uint8) {
 
 // tileMapAddress returns the byte address in the VRAM of the tile we are looking for in the tilemap
 // See here: https://wiki.superfamicom.org/backgrounds
-func (bg *bg) tileMapAddress(x uint16, y uint16) uint16 {
-	// TODO: verify that, not sure at all about this
```

Comment:
So is it the correct formula for getting the tileMapAddress ?

Reply:
It seems to be yes